ngraph/frontend/onnx_import/src/op/dropout.cpp

```diff
@@ -0,0 +1,137 @@
+                    }
+                    else
+                    {
+                        return {input_data};
+                    }
+                }
+            }
+
+            namespace set_12
+            {
+                OutputVector dropout(const Node& node)
+                {
+                    const auto ng_inputs = node.get_ng_inputs();
+                    // seed attribute is ignored because traning mode is not supported anyway
+
+                    // default values of inputs
+                    double ratio = 0.5f;
+                    bool training_mode = false;
+
+                    if (ng_inputs.size() > 1)
+                    {
+                        if (!ngraph::op::is_null(ng_inputs.at(1)))
+                        {
+                            CHECK_VALID_NODE(
+                                node,
+                                ngraph::op::is_constant(ng_inputs.at(1).get_node_shared_ptr()),
+                                "Not constant (or omitted) ratio input is not supported.");
+                            ratio = as_type_ptr<default_opset::Constant>(
+                                        ng_inputs.at(1).get_node_shared_ptr())
+                                        ->cast_vector<double>()[0];
+                        }
+                    }
+                    if (ng_inputs.size() > 2)
+                    {
+                        if (!ngraph::op::is_null(ng_inputs.at(2)))
+                        {
+                            CHECK_VALID_NODE(
+                                node,
+                                ngraph::op::is_constant(ng_inputs.at(2).get_node_shared_ptr()),
+                                "Not constant (or omitted) training_mode input is not supported.");
+                            training_mode = as_type_ptr<default_opset::Constant>(
+                                                ng_inputs.at(2).get_node_shared_ptr())
+                                                ->cast_vector<bool>()[0];
+                        }
+                    }
+                    return build_dropout(node, ratio, training_mode);
+                }
+            } // namespace set_12
+
+            namespace set_7
+            {
+                OutputVector dropout(const Node& node)
+                {
+                    // "is_test" attribute was removed
+                    const bool training_mode = false;
+                    const auto ratio = node.get_attribute_value<float>("ratio", 0.5f);
+
+                    return build_dropout(node, ratio, training_mode);
+                }
+            } // namespace set_7
+
+            namespace set_1
+            {
+                OutputVector dropout(const Node& node)
+                {
+                    CHECK_VALID_NODE(node,
+                                     !node.has_attribute("consumed_inputs"),
+                                     "Legacy consumed_inputs attrubute is not supported.");
```

Comment:
Can we just ignore it?

Reply:
ok it makes sense, I leaved only comment in the code